plugins/pulp_rpm/plugins/importers/yum/parse/rpm.py

```diff
@@ -67,38 +69,12 @@ def change_location_tag(primary_xml_snippet, relpath):
     start_index = primary_xml_snippet.find("<location ")
     end_index = primary_xml_snippet.find("/>", start_index) + 2  # adjust to end of closing tag
 
-    first_portion = string_to_unicode(primary_xml_snippet[:start_index])
-    end_portion = string_to_unicode(primary_xml_snippet[end_index:])
-    location = string_to_unicode("""<location href="%s"/>""" % (
-        file_utils.make_packages_relative_path(relpath)))
+    first_portion = primary_xml_snippet[:start_index]
```

Comment:
I think the doc block above needs to be changed to reflect that it takes ``unicode`` in and returns ``unicode`` back.